### DISCUSSION TITLE
assessed income and debt endpoints return internal server error

### DISCUSSION THREAD

**Discussion by inertia666** [DevSkillsHQ/discusssions#25] [Help] [answered]
access-control-allow-origin: *  age: 0  cache-control: public,max-age=0,must-revalidate  content-length: 21  date: Wed,10 May 2023 06:19:36 GMT  server: Vercel  strict-transport-security: max-age=63072000  x-firefox-spdy: h2  x-matched-path: /api/credit-data/debt/[ssn]  x-vercel-cache: MISS  x-vercel-id: arn1::iad1::krm8v-1683699576201-b359b1117e36 
 
 These income/debt enpoints don't work. I get an internal server error.

**Comment by fenske** [comment]
Hi @inertia666. I'm Anton from Alva. Happy to help. There is some ongoing maintenance of the underlying infra affecting the endpoint. Sorry for the inconvenience. I'll let you know once it's addressed.

**Comment by fenske** [comment, marked as answer]
Hi again @inertia666. This has been fixed. Sorry again about the issue.

I wish you the best of luck with the rest of your interview process!

Best,
Anton.

**Comment by inertia666** [comment]
Yes it's working.

I have finished the assignment (Backend, C#) and it runs locally and passes the NodeJS e2e tests but the pipeline is not able to contact the server.

I have not changed anything with the webserver setup in the project. 

Please help me asap so I can get it working and create the PR.

On May 10, 2023 at 9:24 AM, Anton Fenske ***@***.***> wrote:

Closed #25 as resolved.

[a comment by an automated account is not shown]

**Comment by inertia666** [comment]
I changed that and now it just says "no tests were run".

Not really sure what I am supposed to do here.

On May 10, 2023 at 4:56 PM, Alva Robot ***@***.***> wrote:

It seems like you need to update the port to 5000 here as the test runner is currently hitting a different port than the one your app is running on.

**Reply by fenske** [reply]
GitHub has an ongoing major incident that prevents GitHub Actions from spinning up. https://www.githubstatus.com/

I can re-trigger your pipeline once it's recovered.

**Reply by fenske** [reply]
I realize it's unfortunate that you've hit tech issues twice with our providers (a really annoying coincidence). Once again, I'm so sorry about that. 

But given your tests pass locally, there shouldn't be any more action to do on your side (as I mentioned above, I'll make sure to re-trigger the tests once GitHub is back up).

BR,
Anton.

**Reply by fenske** [reply]
The build passes now. ✅ Let me know if you have any other questions. Best of luck with the rest of your interview process!

**Comment by inertia666** [comment]
Thanks, that's great.

I have created the PR and commented a few things I want to talk about. Thought process/observations are included in the main commit.

I am not really that familar with Githubs review process so i wasn't sure if i had to submit anything to indicate my review comments are finished.

Anyway, I am all done now so please pass on the results to NovaCura.

/Adam

On May 11, 2023 at 7:48 AM, Anton Fenske ***@***.***> wrote:

The build passes now. ✅ Let me know if you have any other questions. Best of luck with the rest of your interview process!
